helmfile.d/snippets/defaults.yaml

```diff
@@ -94,6 +94,7 @@ environments:
             _rawValues: {}
           cert-manager:
             issuer: custom-ca
+            externallyManagedTlsSecretName: otomi-byo-wildcard-cert
```

Comment:
I wouldn't set a default for this and also not use it in the secret for byo-wildcard-cert. Because this will end up in the values repository even if you don't use the byo-wildcart-cert. That can cause confusion to the end user.

Reply:
Currently it is written in the non-functional requirements that the `xternallyManagedTlsSecretName` should refer to the existing hard-coded secret name.

We could also name things differently, but technically the only difference between the `byo-wildcard-cert` and `externally-managed-tls-secret` is, that the certificate data of the former is stored in the values repo. They both have in common that renewal remains at the responsibility of the platform admin or an external automation.

There are however reasons to keep the concepts strictly separated. For example, when the secret exists and one migrates from `byo-wildcard-cert` to `externally-managed-tls-secret`, it can happen that the deployment process removes or overwrites a secret that was stored in the cluster.

Reply:
Reverted this change.